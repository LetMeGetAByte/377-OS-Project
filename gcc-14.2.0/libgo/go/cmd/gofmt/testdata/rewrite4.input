version https://git-lfs.github.com/spec/v1
oid sha256:fdf35a5848c462452b4c06c00b964e7486d69ccded3e1da0f8f4a241d52bcd39
size 1149
