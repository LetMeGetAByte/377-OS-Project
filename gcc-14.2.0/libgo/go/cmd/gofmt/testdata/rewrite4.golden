version https://git-lfs.github.com/spec/v1
oid sha256:4c8caf156805ac97e258f5dc1aa09282677aa93c38eb44b91e9e52fcbf3f9bac
size 1063
